### ISSUE TITLE
Detainers should remain in the ~/Detainer directory

### ISSUE THREAD

**Issue by TheNightmanCodeth** [TheNightmanCodeth/Detainer#2]
I think the default usecase should be for average users who are unfamiliar with the way gocryptfs actually works. A user should be able to add folders and files to a detainer from the UI. The files and folders should be able to be opened from the UI as well. 

In other words, a custom location should not even be a possibility until the rest of the UI is completely finished, and even then it should be hidden in some advanced settings.

**Comment by arshubham** [comment]
I agree custom directory cam be added later. But please dont't add the directory directly to user home. The hidden dot files create a mess as it is. Better not have an actual directory in the home. Better solution will be to keep it in documents by default.

**Comment by TheNightmanCodeth** [comment]
@arshubham Documents sounds about right but I wonder if it really makes sense since files other than "documents" can be stored inside of a detainer as well. I'd like to hear more feedback from the community before I change the default but I'll get the settings for it implemented asap
